html/editing/editing-0/contenteditable/contentEditable-shortcuts.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Editing: manual test for keyboard shortcuts with contenteditable=plaintext-only</title>
+    <link rel="help" href="https://html.spec.whatwg.org/multipage/#contenteditable"/>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+
+    <div id="contentEditablePlaintextOnly" contenteditable="plaintext-only">plaintext-only</div>
+</head>
+<body>
+<script>
+    var kControl = "\uE009";
+    var kMeta = "\uE03D";
+    var kB = "B";
+
+    promise_test(async () => {
+        var contentEditablePlaintextOnly = document.getElementById('contentEditablePlaintextOnly');
+
+        var selection = document.getSelection();
+        selection.selectAllChildren(contentEditablePlaintextOnly);
+
+        await new test_driver.Actions()
+            .keyDown(kControl)
```

Comment:
I'm not familiar with this function but it looks like this sends `Ctrl+B`. Does this work fine on Mac, where the bold shortcut is `Cmd+B`?